### PR TITLE
Refactor quarto.nbdev_readme; add docs

### PR DESCRIPTION
**Pull request by hugetim**
For context: https://forums.fast.ai/t/nbdev-v2-codebase-falls-short-of-literate-programming/103671/7

### CODE REVIEW

File: nbs/api/14_quarto.ipynb [hunk 1]
```diff
@@ -11,6 +11,14 @@
     "- order: 14"
```
Comment (jph00) [inline nbs/api/14_quarto.ipynb:11]: <pre class="ql-syntax" spellcheck="false">Line #3.    class SidebarYmlRemoved:
</pre><p>This should be private, so its name should start with <code>_</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (jph00) [inline nbs/api/14_quarto.ipynb:11]: <pre class="ql-syntax" spellcheck="false">Line #4.        def __init__(self,path): self._path=path
</pre><p>Even although this is private a single line docstring saying what it's for might help people understand the code better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (jph00) [inline nbs/api/14_quarto.ipynb:11]: <p>I'd remove this - it takes up a lot of room in the docs and dupes what can be found in the link you provide above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (jph00) [inline nbs/api/14_quarto.ipynb:11]: <p>Can you remove the output here please? (Just re-run the cell when it's a comment and the output will disappear.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

**Comment by jph00** [comment]
Many thanks! I've added some very minor suggestions.

**Comment by jph00** [comment]
Ready to look at it @hugetim ?

**Comment by hugetim** [comment]
I think so. My only hesitation is that I haven't looked at your comments with ReviewNB yet, only within the GitHub gui. But I think your meaning was clear.

File: nbs/api/14_quarto.ipynb (hunk 1 above)
Comment (hugetim) [reply nbs/api/14_quarto.ipynb:11]: <p>Any preference on whether the docstring is for the class or the init method? </p>

**Comment by hugetim** [comment]
<p>Ok, I was able to look at ReviewNB to be sure. I think it's ready, but I did add one question re the docstring suggestion.</p>

**Comment by hugetim** [comment]
And then I made a minor wording change to try to make the chk_time line more quickly legible. All ready, now, @jph00.

File: nbs/api/14_quarto.ipynb (hunk 1 above)
Comment (jph00) [reply nbs/api/14_quarto.ipynb:11]: <p>Generally IIRC we put it on the class.</p>

**Comment by jph00** [comment]
Many thanks @hugetim ! :) Feel free to add the missing docstring in a separate PR if you're OK with with.

**Pull request merged**